Tasks/ManualIntervention/task.json

```diff
@@ -38,6 +38,18 @@
             "defaultValue": "",
             "required": false,
             "helpMarkDown": "Send a manual intervention pending email to specific users (or groups). Only users with manage deployment permission can act on a manual intervention."
+        },
+        {
+            "name": "onTimeout",
+            "type": "radio",
+            "label": "On timeout",
+            "defaultValue": "Reject",
+            "required": true,
```

Comment:
make it an optional parameter and file a bug on task editor team to show the default value in the UI for optional parameters